monstache.go

```diff
@@ -82,7 +82,7 @@ var chunksRegex = regexp.MustCompile("\\.chunks$")
 var systemsRegex = regexp.MustCompile("system\\..+$")
 var exitStatus = 0
 
-const version = "6.7.17"
+const version = "6.7.10+4"
```

Comment:
I am just curious about this; do we really have a version naming convention like 6.7.10+4?